src/backend/index.js

```diff
@@ -70,24 +70,24 @@ function processFeeds(feeds) {
 }
 
 /**
- * Download and parse feed author/URL info from the wiki, and process
+ * Download and parse feed author/URL info from the wiki and redis, and process
  * these into Feed Objects to be added to the database and feed queue.
  */
-async function processWikiFeeds() {
+async function processAllFeeds() {
   try {
-    // Get an Array of Feed objects from the wiki feed list
-    const feeds = await getWikiFeeds();
+    // Get an Array of Feed objects from the wiki feed list and Redis
+    const feeds = await Promise.all([...Feed.all(), ...getWikiFeeds()]);
```

Comment:
This is erroring for me, because `Feed.all()` returns a Promise with an Array, but not an Array of Promises.  It works if you do this:

```js
// Get an Array of Feed objects from the wiki feed list and Redis
const [all, wiki] = await Promise.all([Feed.all(), getWikiFeeds()]);
// Process these feeds into the database and feed queue
await processFeeds([...all, ...wiki]);
```

Reply:
> Tested this locally, and it throws.

Strange, I didn't notice anything when I ran `npm test`... 🤔 

> It works if you do this:
> 
> ```js
> // Get an Array of Feed objects from the wiki feed list and Redis
> const [all, wiki] = await Promise.all([Feed.all(), getWikiFeeds()]);
> // Process these feeds into the database and feed queue
> await processFeeds([...all, ...wiki]);
> ```

Change made in https://github.com/Seneca-CDOT/telescope/commit/cd485162fa7f7ec4798be39868c367bc10898133; thanks!